### ISSUE TITLE
contact!

### ISSUE THREAD

**Issue by bagder** [microsoft/MonocleAI#4]
Hi,

I can't contact you on the given email address as it claims:

> "Your message to the group monocleai@microsoft.com couldn't be delivered. The group monocleai only accepts messages from people on its allowed senders list, and your email address isn't on the list."

... which is a bit frustrating since you emailed me with that address in `From:`. This leaves me no way to contact you about this bot.

**Comment by monocle-ai** [comment]
Hi,

Thank you for reaching out to us.
We looked into this issue and it seems like we had some settings issue which restricted anyone from outside of Microsoft to send an email. It takes 3-4 days for the permissions to reflect in the system. But until then we can discuss any of the your concerns here.

Also, we did trigger the code analysis pipeline for curl and the reason it didn't trigger in the first place was that we require a PR to be merged to master to formally kick start the process. Having said that we also added the functionality of triggering the code analysis pipeline to file and fix issues even at the time of app installation.

Thanks,
Team MonocleAI
